portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,37 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
+import com.google.gson.Gson;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/greeting")
+@WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+  private int curIndex;
```

Comment:
What is the purpose of curIndex?

Reply:
Oops, left over code from when I was trying out something.

Reply:
Removed unnecessary code